conf/application.conf

```diff
@@ -46,7 +46,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 # Custom error handler
 play.http.errorHandler = "uk.gov.hmrc.plasticpackagingtax.returns.config.ErrorHandler"
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com data:"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com www.googletagmanager.com tagmanager.google.com data:"
```

Comment:
This is already in master. Some weird rebase issues:
https://github.com/hmrc/plastic-packaging-tax-returns-frontend/blob/master/conf/application.conf#L49

Reply:
yeah not sure why, shouldn't loose anything though.